src/Form/Element/PhoneNumber.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\I18n\PhoneNumber\Form\Element;
+
+use Laminas\Form\Element;
+use Laminas\I18n\CountryCode;
+use Laminas\I18n\PhoneNumber\PhoneNumberValue;
+use Laminas\I18n\PhoneNumber\Validator\PhoneNumber as NumberValidator;
```

Comment:
I would prefer an exact name here, because when I read the code it was not clear to me that it is the phone number validator of this package.

```suggestion
use Laminas\I18n\PhoneNumber\Validator\PhoneNumber as PhoneNumberValidator;
```

Reply:
Done! Thanks @froschdesign